src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,91 @@
+package duke;
+import java.util.ArrayList;
+
+/**
+ * Encapsulates a list of Tasks.
+ */
+class TaskList {
+    /** An ArrayList of Tasks.*/
+    ArrayList<Task> tasks;
+
+    /**
+     * Constructs a new TaskList.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+    /**
+     * Constructs a new TaskList.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Lists all the tasks stored.
+     */
+    public void listTasks() {
+        String topDivider = "~~~~~~~~~~~~~~~~o~~~~~~~~~~~~~~~~\n" + "Duke's Response: \n"
+                    + "Here are the tasks in your list: \n";
+        String botDivider = "~~~~~~~~~~~~~~~~o~~~~~~~~~~~~~~~~";
+        System.out.println(topDivider);
```

Comment:
Should this be stored under the Ui class?

Reply:
Yes, that is a good suggestion. Have done that already. @thennant